lib/helpers.bash

```diff
@@ -23,7 +23,13 @@ function _command_exists ()
   _example '$ _command_exists ls && echo exists'
   _group 'lib'
   local msg="${2:-Command '$1' does not exist!}"
-  type "$1" &> /dev/null || (_log_warning "$msg" && return 1) ;
+  if type -t "$1" &> /dev/null
```

Comment:
I think this additional `-t` is either a typo, or should be left for a different PR. However, I'm totally open to hearing out your reasoning.

Unless I'm mistaken, `-t` check for an open file descriptor.
```plain
-t FD  file descriptor FD is opened on a terminal
```

Reply:
`type -t` asks what the type of the command is: function, alias, file, builtin, &c. This reduces the amount of output, even though it's send directly to /dev/null.

Reply:
@gaelicWizard indeed i am mistaken. thanks